### DISCUSSION TITLE
How to upgrade freom version 4.* to 5.* ?

### DISCUSSION THREAD

**Discussion by a3adel** [GetStream/stream-chat-android#4480] [Q&A]
I have an older version and when I'm trying to upgrade to the latest its really hard, there is no upgrading documentation and the Deprecation.md doesn't have much of the removed and deprecated functions and files

**Comment by adasiewiczr** [comment]
Hey @a3adel,
you can find the migration guide [here](https://getstream.io/chat/docs/sdk/android/client/guides/chatdomain-migration/).
Let us know if you have any further questions.

**Reply by a3adel** [reply]
thanks @adasiewiczr

**Comment by a3adel** [comment]
What about other classes like `Member` class it has different roles like `admin` and `lastActive` extenstion function
how can I get those now ?
